--- a/build.zig
+++ b/build.zig
@@ -11,14 +11,7 @@
     // between Debug, ReleaseSafe, ReleaseFast, and ReleaseSmall.
     const optimize = b.standardOptimizeOption(.{});
 
-    const use_openssl = b.option(bool, "openssl", "Use system-installed openssl for TLS support in zap") orelse blk: {
-        // Alternatively, use an os env var to determine whether to build openssl support
-        if (std.process.getEnvVarOwned(b.allocator, "ZAP_USE_OPENSSL")) |val| {
-            defer b.allocator.free(val);
-            if (std.mem.eql(u8, val, "true")) break :blk true;
-        } else |_| {}
-        break :blk false;
-    };
+    const use_openssl = b.option(bool, "openssl", "Use system-installed openssl for TLS support in zap") orelse false;
 
     const facilio = try build_facilio("facil.io", b, target, optimize, use_openssl);
 
@@ -29,6 +22,8 @@
     });
     zap_module.linkLibrary(facilio);
 
+    if (true) return;
+
     const all_step = b.step("all", "build all examples");
 
     // -- Docs
